cabal.project

```diff
@@ -14,8 +14,7 @@ repository cardano-haskell-packages
 -- you need to run if you change them
 index-state:
   , hackage.haskell.org 2024-04-15T08:36:34Z
```

Comment:
FYI, uually, when we bump CHaP it's good practice to bump hackage too, in case packages recently added to CHaP rely on recent version of hackage. Or so I was told. Approving anyway :slightly_smiling_face:

Reply:
Actually, it's the opposite 😄 It's better when CHaP is newer than hackage.